components/LandingPage.tsx

```diff
@@ -2,97 +2,216 @@ import * as React from 'react'
+              Our learning process is interactive and follows the same practice
+              as a well functioning engineering team. From the first line of
+              code you write, you will be code reviewed by engineers to ensure
+              you are always following industry best practices.
+            </h4>
+          </div>
+        </div>
+        <div className="row mt-5 align-items-start mb-5">
+          <div className="col-md-4">
+            <img
+              src="choose-01.svg"
+              alt="Curriculum"
+              style={{ height: '170px' }}
+            ></img>
+            <h4 className="mt-3 font-weight-bold">Modern Curriculum</h4>
+            <p className="mt-4 font-weight-light">
+              First the curriculum will help you achieve mastery in JavaScript.
+              Using JavaScript, you will learn about different branches of
+              software engineering: websites, mobile applications, servers, and
+              machine learning.
+            </p>
+          </div>
+          <div className="col-md-4">
+            <img
+              src="choose-02.svg"
+              alt="Support"
+              style={{ height: '170px' }}
+            ></img>
+            <h4 className="mt-3 font-weight-bold">Support</h4>
+            <p className="mt-4 font-weight-light">
+              One of the most important skills in a challenging workplace is
+              knowing how to ask for help. Our curriculum is intentionally
+              difficult and we have dedicated mentors to answer your questions
+              and guide you how to communicate effectively.
+            </p>
+          </div>
+          <div className="col-md-4">
+            <img
+              src="choose-03.svg"
+              alt="Curriculum Image"
+              style={{ height: '170px' }}
+            />
+            <h4 className="mt-3 font-weight-bold">Work Experience</h4>
+            <p className="mt-4 font-weight-light">
+              Once you have completed our curriculum and learned to communicate
+              effectively, you will be invited to contribute to this open-source
+              project to improve the learning experience for future students.
+            </p>
+          </div>
         </div>
       </div>
-      <div className="row mt-5">
-        <div className="col-md-10 offset-md-1">
-          <h4 className="font-weight-light" style={{ lineHeight: 1.5 }}>
+      <div
+        className="container-fluid py-5"
+        style={{ background: 'rgb(84,64,216)', color: 'white' }}
+      >
+        <div className="container mt-5">
+          <h1
+            id="journey"
+            className="mt-5 display-4 font-weight-bold text-center"
+          >
+            Your Learning Journey
+          </h1>
+          <h4
+            className="mt-5 font-weight-light col-md-8 offset-md-2 pb-5 text-center"
+            style={{ lineHeight: 1.5 }}
+          >
```

Comment:
I think this would be a <p> instead of <h4>, it seems more like a paragraph than a heading.

Reply:
I changed it to a p and I added a `font-size` as inline style. Please let me know if there's any better way to do this.

Reply:
Since we're using Bootstrap, you can use `<p className="h4">` if you want the font-size to be the same size.

Source: https://getbootstrap.com/docs/4.0/content/typography/#headings

Reply:
Thanks! I was looking for that section...